src/Orleans.Core/Statistics/ClientStatisticsManager.cs

```diff
@@ -83,17 +86,27 @@ internal class ClientStatisticsManager : IDisposable
                     logStatistics.StatsTablePublisher = statsProvider;
                     // Note: Provider has already been Init-ialized above.
```

Comment:
I couldn't spot where it's initialized above

Reply:
it is not above anymore, IStatisticsPublisher are now being initialized as a `IProvider`. Take a look at `LegacyProviderConfigurator.COnfigureServices` to see how it is configured using the legacy way. 

In the new way, user will need to configure it with DI as  `IProvider` and then register it into lifecycle.

Reply:
but the xml is probably wrong at this point.